docs/stability.md

```diff
@@ -0,0 +1,50 @@
+# OSBA Versions and Module Stability
+
+OSBA has an overall stability, defined by its release version, which is
+based on [semver](https://semver.org/), but each of its
+[services](/README.md#supported-services) also has a stability label.
+
+# Overall Stability and OSBA Version
+
+We version OSBA generally via its release version to indicate its overall
+stability. Generally, this includes how well it's tested, its stability in the wild,
+how many people using it, bug reports, and more.
+
+But when we increase a semver MAJOR, MINOR or PATCH version, we also take into
+account if any modules change in stability. Read on for how this works.
+
+# Service Stability
+
+We indicate a stability for each service in the OSBA catalog and docs to
+indicate how mature it is. Each service may have a different stability:
+
+* `experimental` - We have an idea for a new service that we want to support,
```

Comment:
I'm confused by these definitions. I _thought_ that when we talk about service stability with regards to OSBA we are referring to the actual Azure Service's advertised stability. We aren't going to call a new service offering stable.

Reply:
Good point - I'll make it more clear in here